core/internal/api/send.go

```diff
@@ -0,0 +1,31 @@
+package api
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/hashicorp/go-retryablehttp"
+)
+
+// Sends an HTTP request to the W&B backend.
+func (client *Client) Send(req *Request) (*http.Response, error) {
```

Comment:
what do you think about `MakeRequest`?

Reply:
it sounds like it creates a new Request value. I'd be open to `SendRequest`, but I prefer `Send` so that the usage reads like `Send(request)` rather than `SendRequest(request)`. Go's http library uses `Do`, which is also reasonable, but I usually think of this action as sending a request rather than doing a request.

Reply:
Yeah, valid.  I can see how make request sounds like the creation of a request and make is just weird.